RecoTauTag/Configuration/python/HPSPFTaus_cff.py

```diff
@@ -848,7 +883,71 @@
     hpsPFTauDiscriminationByTightMuonRejection3,
     hpsPFTauVertexAndImpactParametersTask,
     hpsPFTauMVAIsolation2Task
-)
+    )
+
 produceAndDiscriminateHPSPFTaus = cms.Sequence(
     produceAndDiscriminateHPSPFTausTask
-)
+    )
+
+hpsPFTauDiscriminationByVVLooseIsolationMVArun2v1DBoldDMwLT = hpsPFTauDiscriminationByVLooseIsolationMVArun2v1DBoldDMwLT.clone()
+hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTTaskExt = hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTTask.copy()
```

Comment:
please use a leading underscore here so that this module does not propagate downstream for ```import *```

```_hpsPFTauDiscriminationByIsolationMVArun2v1DBoldDMwLTTaskExt```